docs/source/examples/dataframe-hdf5.rst

```diff
@@ -0,0 +1,111 @@
+
+  **Important Note** -- `h5py <http://docs.h5py.org/en/latest/index.html>`_ and `PyTables <http://www.pytables.org/index.html>`_ are the two premiere libraries to work with HDF5 data in Python.   ``dask.dataframe``, `like python pandas <http://pandas.pydata.org/pandas-docs/stable/io.html#io-hdf5>`_, uses the the `PyTables <http://www.pytables.org/FAQ.html#what-is-pytables>`_ library to interface with the HDF5 binary data format. HDF5 files made using the h5py project may not read into a dask DataFrame. 
+
+  While there is `planned work to refactor PyTables to use h5py bindings to HDF5 and create 100% compatibility <https://groups.google.com/forum/#!msg/pytables-users/GBVkvGexSag/IwLjEtIfKIgJ>`_, there are differences in how the interfaces are implemented. `dask.dataframe` will have `nearly seamless compatibility with HDF5 files built in pandas <https://github.com/bokeh/bokeh/issues/1246>`_.  Here is more information on the differences between *h5py* and *PyTables* and work to improve compatibility:
+
+    -  `h5py explanation of differences from PyTables <http://docs.h5py.org/en/latest/faq.html#what-s-the-difference-between-h5py-and-pytables>`_
+    -  `PyTables explanation of differences from h5py <http://www.pytables.org/FAQ.html#how-does-pytables-compare-with-the-h5py-project>`_
+    -  `HackFest with h5py and PyTables teams <https://curtinic.github.io/python-and-hdf5-hackfest/>`_
+
+**Core DataFrame Functions in this Tutorial:**
+----------------------------------------------
+
+    *  Use ``dask.dataframe`` to:
+        
+        1.  Create dask DataFrame by loading a specific dataset (key) from a single HDF5 file
+        2.  Read all datasets from a single HDF5 into a dask DataFrame 
+        3.  Create a dask DataFrame by loading multiple HDF5 files with different datasets (keys)
+
+
+**Generate Synthetic HDF5 files:**
+--------------------------------------------
+
+These exercises require multiple HDF5 files with multiple dataset keys.  The `Python pandas library <http://pandas.pydata.org/pandas-docs/stable/>`_ has a convenient `I/O API to create an HDF5 file from a dataframe <http://pandas.pydata.org/pandas-docs/stable/io.html#io-hdf5>`_. Combining this method with the `Python numpy random integer generator <http://docs.scipy.org/doc/numpy/reference/generated/numpy.random.randint.html>`_ and several standard library modules provide all the tools necessary to create a synthetic dataset.
+
+.. code-block:: python
+   :emphasize-lines: 40,45
+   
+    
+    def create_hdf5s(iters):
+           
+       import random
+       import pandas as pd
+       import numpy as np
+       import string 
+       import json
+    
+       # Intitalize counter to build specific number of hdf5 files
+       count = 0
+       setseed=1
+       np.random.seed(seed=setseed)
+       random.seed(setseed)
```

Comment:
Can we drop some of these in order to save space?  This example is long and not to the point of a reader's question of "how do I read hdf files" so I'd like to keep it as small as possible